libraries/AP_NavEKF2/AP_NavEKF2_Measurements.cpp

```diff
@@ -403,16 +392,12 @@ void NavEKF2_core::readIMUData()
 
 // read the delta velocity and corresponding time interval from the IMU
 // return false if data is not available
-bool NavEKF2_core::readDeltaVelocity(uint8_t ins_index, Vector3f &dVel, float &dVel_dt) {
+void NavEKF2_core::readDeltaVelocity(uint8_t ins_index, Vector3f &dVel, float &dVel_dt) {
```

Comment:
I'd keep the failure. If all IMUs fail then I think it can still fail

Reply:
Also the comment on line 394 does not match the code